src/style.css

```diff
@@ -0,0 +1,83 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+body {
+  box-sizing: border-box;
+}
+
+.search-bar {
+  position: relative;
+  margin: 20px auto;
+}
+
+.image-big {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  margin-top: 25px;
+  margin-left: 26px;
+}
+
+.image-small {
+  position: absolute;
+  width: 11px;
+  height: 11px;
+  margin-top: 15px;
+  margin-left: 13px;
+
+}
+
+.big-bar,
+.small-bar {
+  width: 100%;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  font-family: "Avenir", sans-serif;
+  color: #3d4e61;
+}
+
+.big-bar {
+  height: 70px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  font-size: 16px;
+  line-height: 22px;
+  padding-left: 62px;
+}
+
+.small-bar {
+  box-sizing: border-box;
+  height: 42px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  font-style: normal;
+  font-size: 14px;
+  line-height: 19px;
+  padding-left: 33px;
+}
+
+.big-bar:hover,
+.small-bar:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.big-bar:focus,
+.small-bar:focus {
+  font-weight: 900;
```

Comment:
need to remove the blue oultine on focus 😉

<img width="181" alt="image" src="https://user-images.githubusercontent.com/55802630/172384350-5af4a648-8dd6-400d-be97-eeefcb465ff8.png">

Reply:
ok